batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/metadata.json

```diff
@@ -26,5 +26,7 @@
         "5.6",
         "5.8",
         "6.0"
-    ]
-}
+    ],
+    "author": "pdcurtis",
+    "last-edited": 1704578690
```

Comment:
The `last-edited` key value shouldn't be here. This will fail validation.

Reply:
no idea how that got there tbh